src/pages/LenderProfile/LenderProfile.vue

```diff
@@ -33,6 +43,55 @@ export default {
 		LenderProfileWrapper,
 		NotFoundWrapper,
 	},
+	setup() {
+		const apollo = inject('apollo');
+
+		const {
+			isBadgeKeyValid, fetchAchievementData, fetchContentfulData, badgeData, getTierBadgeDataByLevel
+		} = useBadgeData();
+
+		const route = useRoute();
+
+		const utmCampaign = route.query?.utm_campaign ?? '';
+		const badgeLevel = route.query?.badge_level ?? 0;
+		const badgeKey = utmCampaign.split('badge_')[1];
+		const enableBadgeContent = ref(false);
+
+		onMounted(async () => {
+			if (isBadgeKeyValid(utmCampaign)) {
```

Comment:
This data used to be queried server-side, but it's now clientside in mounted. Is that an issue?

Reply:
Ah yes, we do need the page title populated on server render.